WTWSMS/common/cultures/00_cultures.txt

```diff
@@ -4644,7 +4645,13 @@ byzantine = {
 		
 		founder_named_dynasties = yes
 
-		modifier = default_culture_modifier
+		allow_looting = yes
+
+		character_modifier = {
+			martial = 4
```

Comment:
The Character Modifiers applies to those of the Spartan culture, and _only_ to the Spartan culture. There is supposed to be a long and arduos trial to become Spartan, so the Character Modifiers are only added after the journey is finished.